app/models/redis_endpoint.rb

```diff
@@ -0,0 +1,19 @@
+class RedisEndpoint < Endpoint
+  store :options, accessors: [:namespace]
+
+  def switch!
+    Sufia.config.redis_namespace = options[:namespace]
+  end
+
+  def reset!
+    redis_instance.keys.each do |namespaced_key|
```

Comment:
What does this method do? Our other endpoints reset the connection to the non-account-namespaced form, but it seems like this might do more?